qa/integration-tests/src/test/java/io/camunda/zeebe/it/gateway/GatewayAuthenticationIdentityIT.java

```diff
@@ -109,6 +112,7 @@ public class GatewayAuthenticationIdentityIT {
   private static final ZeebeContainer ZEEBE =
       new ZeebeContainer(ZeebeTestContainerDefaults.defaultTestImage())
           .withNetwork(NETWORK)
+          .dependsOn(KEYCLOAK, IDENTITY)
```

Comment:
No, as of now the identity-sdk is not reaching out to keycloak if there is is no request to handle yet. However this might change at some point and I figured it's generally fine waiting until the auth infrastructure is fully up, just in case behavior changes.